pyproject.toml

```diff
@@ -107,11 +107,11 @@ features = ["dev"]
 [tool.hatch.envs.default.scripts]
 lint = [
   "ruff check src/ tests/ examples/",
-  "black --check src/ tests/ examples/",
+  "black --check docs/notebooks",
```

Comment:
The Ruff linter supports notebooks, see: https://docs.astral.sh/ruff/faq/#does-ruff-support-jupyter-notebooks

Did you try to use that to format the notebooks as well or not?